apps/test/unit/studio/itemTest.js

```diff
@@ -11,32 +16,79 @@ import {
   BEHAVIOR_GRID_ALIGNED
 } from '@cdo/apps/studio/constants';
 
-import Item from '@cdo/apps/studio/Item';
-
 describe('item', () => {
   before(() => {
     Studio.trackedBehavior = {};
     Studio.trackedBehavior.createdItems = {};
+    Studio.SQUARE_SIZE = 50;
   });
 
   let item;
 
   describe('update', () => {
+    describe('if destination is not yet set', () => {
+      before(() => {
+        // the destination-setting logic uses _.shuffle to semi-randomize the
+        // set of possible destinations. Stub that with something deterministic
+        // to make testing easier
+        sinon.stub(_, 'shuffle').callsFake(ar => ar);
+      });
+
+      let targetSprite;
+      beforeEach(() => {
+        item = new Item({});
+        item.x = 100;
+        item.y = 100;
+
+        item.hasWall = sinon.stub().returns(false);
+
+        targetSprite = new Sprite({});
+        targetSprite.x = 200;
+        targetSprite.y = 200;
+        Studio.sprite = [targetSprite];
+        item.targetSpriteIndex = 0;
+      });
+
+      it('sets an arbitrary direction on wander', () => {
+        item.activity = BEHAVIOR_WANDER;
+        item.update();
+        expect(item.dir).to.equal(Direction.NORTH);
```

Comment:
Oh, sorry I didn't reply to this.  I'm fine with this approach too.  I just want to make sure the test case calls out how we're using or bypassing randomness.